src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Logging/LogHubLoggerProvider.cs

```diff
@@ -1,70 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT license. See License.txt in the project root for license information.
-
-using System;
-using System.Diagnostics;
-using Microsoft.Extensions.Logging;
-
-namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Logging;
-
-internal class LogHubLoggerProvider : ILoggerProvider
-{
-    // Internal for testing
-    internal const string OmniSharpFrameworkCategoryPrefix = "OmniSharp.Extensions.LanguageServer.Server";
-
-    private readonly LogHubLogWriter _logWriter;
-    private readonly ILogger _noopLogger;
-
-    public LogHubLoggerProvider(LogHubLogWriter logWriter)
-    {
-        if (logWriter is null)
-        {
-            throw new ArgumentNullException(nameof(logWriter));
-        }
-
-        _logWriter = logWriter;
-        _noopLogger = new NoopLogger();
-    }
-
-    public ILogger CreateLogger(string categoryName)
-    {
-        if (categoryName is null)
-        {
-            throw new ArgumentNullException(nameof(categoryName));
-        }
-
-        if (categoryName.StartsWith(OmniSharpFrameworkCategoryPrefix, StringComparison.Ordinal))
```

Comment:
Maybe you would still want to keep this logic

Reply:
We don't have any OmniSharp code in main any more